svthalia/concrexit
# Move user menu to dropdown instead of page

### Summary
This PR moves the user menu from a page to the dropdown menu that already existed. I believe this greatly improves the user experience since this information is now easily accessible and not hidden away anymore.

### How to test
Steps to test the changes you made:
1. Login with a valid user
2. Check the user menu at the top of the page
3. It should be one single button instead of two and the dropdown should contain the items previously found on the UserAccountView page that has been removed.

![Screenshot 2020-03-26 at 13 04 45](https://user-images.githubusercontent.com/1799914/77645063-63cbcc00-6f62-11ea-84fd-bff6130dd8ef.png)
